src/main/java/create_task_use_case/CreateTaskController.java

```diff
@@ -0,0 +1,21 @@
+package create_task_use_case;
+
+/**
+ * Task Creation Controller Class; invokes the implementation of createTask given
+ * by CreateTaskInteractor to execute the application logic.
+ * @author Fardin Faruk
+ */
+public class CreateTaskController {
+
+    private final CreateTaskInputBoundary inputBoundary;
+
+    public CreateTaskController(CreateTaskInputBoundary inputBoundary) {
+        this.inputBoundary = inputBoundary;
+    }
+
+    CreateTaskOutputData createTask(String day, String title){
```

Comment:
Please add a javadoc for this method, and make it public.

Reply:
Done.